var/spack/repos/builtin/packages/hepmc/package.py

```diff
@@ -27,51 +22,16 @@ class Hepmc(CMakePackage):
     version('2.06.05', sha256='4c411077cc97522c03b74f973264b8d9fd2b6ccec0efc7ceced2645371c73618')
 
     variant('python', default=False, description='Enable Python bindings')
-    variant('rootio', default=False, description='Enable ROOT I/O')
-    variant('interfaces', default=False, description='Install interfaces for some Monte-Carlo Event Gens')
 
     depends_on('cmake@2.8.9:', type='build')
     depends_on('python', when='+python')
     depends_on('root', when='+rootio')
```

Comment:
It looks (from the conflicts removed below) like neither of these should be valid dependencies for hepmc when we split out hepmc3

Reply:
good catch, indeed I removed the variants but overlooked those two lines. Fixed.